### DISCUSSION TITLE
求助，怎么改中文名字？

### DISCUSSION THREAD

**Discussion by faatong** [share121/inter-knot#5130] [其他]
怎么显示中文名字？改过了还是显示英文

**Comment by Dave-12138** [comment]
绳网还是github？
- 绳网不显示名字可以刷新试试
- github讨论不显示“Name”只显示“username”是正常现象，github设计如此

<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Reply by faatong** [reply]
绳网，github的登录名只能英文，昵称改成中文了，在绳网还是显示的英文名<img src="https://bbs-static.miyoushe.com/static/2023/11/16/2544d70667e060623d4b7702cd6bb3b9_8763505029750571668.png" alt="可琳-紧张" width="75" >

**Reply by Dave-12138** [reply]
> 绳网，github的登录名只能英文，昵称改成中文了，在绳网还是显示的英文名<img alt="可琳-紧张" width="75" src="https://camo.githubusercontent.com/92ed8bde526fb94f42dbaf34622597c3911dd96315e013085677a0a404cad134/68747470733a2f2f6262732d7374617469632e6d69796f757368652e636f6d2f7374617469632f323032332f31312f31362f32353434643730363637653036303632336434623737303263643662623362395f383736333530353032393735303537313636382e706e67">

![Screenshot_20240918_020804_com.microsoft.emmx.canary.jpg.webp](https://s2.loli.net/2024/09/18/haFYR4Z2ADzjCqN.webp)
刷新试试吧，我这里能显示<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Reply by faatong** [reply]
ok！感谢大佬<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/b6352ebdefa5f62ac25ccc5317fe832a_7207836849676334708.png" alt="停云 暴富" width="75" >
